### DISCUSSION TITLE
`Error: This module was already loaded, so it can't be configured using "with"`

### DISCUSSION THREAD

**Discussion by brandon-rusk** [uswds/uswds#6218] [Q&A] [answered]
Hello,

I believe this is a simple configuration mistake I'm making and I've attempted to follow the documentation. Based on the research I've done, this error is usually tied to a variable being incorrectly named from USWDS. I confirmed that `$theme-show-notifications` does exist in https://designsystem.digital.gov/documentation/settings/#general-settings-2
I am receiving:
``` Error: This module was already loaded, so it can't be configured using "with".
  ┌──> app/_styles/main.scss
9 │ ┌ @use "uswds-core" with (
10│ │   $theme-show-notifications: false,
11│ │ );
  │ └─^ new load
  ╵
  ┌──> app/_styles/components/_test.scss
1 │   @use "uswds-core" as *;
  │   ━━━━━━━━━━━━━━━━━━━━━━ original load
```

I am building using the CLI:
```
sass --load-path=node_modules/@uswds/uswds/packages ../app/_styles/main.scss main.css
```
I also have this configuration in my `next.config.js`:
```
/** @type {import('next').NextConfig} */
 
const nextConfig = {
  sassOptions: {
            includePaths: [
              "./node_modules/@uswds/uswds/packages",
        ],
    },
}

module.exports = nextConfig;
```

My sass entry point:
```
@use 'base';
@use 'components';
@use 'layout';
@use 'pages';
@use 'themes';
@use 'vendors';

@use "uswds-core" with (
  $theme-show-notifications: false,
);

@forward 'uswds';
@forward 'uswds-theme-custom-styles';
```

Thank you to anyone who is able to assist with this!

**Comment by amyleadem** [comment, marked as answer]
Hello @brandon-rusk,
Thanks for reaching out. It looks like this might be related to the order in which you are loading files in your Sass entry point. Please note that I am making a couple of assumptions about your file structure here, so let me know if any of this is inaccurate. 

Based on your error message, it looks like you are calling `@use "uswds-core" as *` inside of `components/_test.scss` file. It also looks like the `components` module is getting loaded before your uswds-core configuration (`@use "uswds-core" with ($theme-show-notifications: false,)`) in your Sass entry point. I believe this is what is causing the error. Are you able to move the `@use "uswds-core" with ()` configuration above the other `@use` rules in your Sass entry point? Wondering if something like this would work for you:

```
@use "uswds-core" with (
  $theme-show-notifications: false,
);

@forward 'uswds';
@forward 'uswds-theme-custom-styles';

@use 'base';
@use 'components';
@use 'layout';
@use 'pages';
@use 'themes';
@use 'vendors';
```

I hope this helps. Please let us know if you have more questions.

**Reply by brandon-rusk** [reply]
It was indeed a simple mistake haha. Thank you!

**Comment by ryparker** [comment]
Changing the Sass configuration order resolved the error, as the author confirmed above. I'm closing this as answered and keeping the [accepted answer](https://github.com/uswds/uswds/discussions/6218#discussioncomment-11373609) available for reference.
